test/functional/wallet_bumpfee.py

```diff
@@ -283,10 +283,10 @@ def test_single_output(self):
         tx = wallet.sendall(recipients=[wallet.getnewaddress()], fee_rate=2, options={"inputs": [utxos[0]]})
 
         # Reduce the only output with a crazy high feerate, should fail as the output would be dust
-        assert_raises_rpc_error(-4, "The transaction amount is too small to pay the fee", wallet.bumpfee, txid=tx["txid"], options={"fee_rate": 1100, "reduce_output": 0})
+        assert_raises_rpc_error(-4, "The transaction amount is too small to pay the fee", wallet.bumpfee, txid=tx["txid"], options={"fee_rate": 1100, "change_output_index": 0})
 
         # Reduce the only output successfully
```

Comment:
We might want to change this comment to
"Change the output index successfully"

Reply:
done, thx